.github/workflows/ci.yaml

```diff
@@ -152,6 +152,7 @@ jobs:
           - { name: "partitioning", case: "partitioning" }
           - { name: "msdos partitioning", case: "msdos" }
           - { name: "debian (amd64)", case: "debian", variables: "-t architecture:amd64" }
+          - { name: "debian with mmdebstrap (amd64)", case: "debian", variables: "-t architecture:amd64 -t tool:mmdebstrap" }
```

Comment:
Also add tests for arm64 and armhf to ensure different archs also work.. Feel free to change the commends to `debian (amd64, debootstrap)` and `debian (amd64, mmdebstrap)` rather then making mmdebstrap special

Reply:
Fixed in latest force push